docs/quick-start/index.md

```diff
@@ -264,10 +273,57 @@ The second field is marked as `(required)` using the custom option imported in
 the `spine/options.proto` file above. This command does not make much sense if there is no text
 to print.
 
+<p class="note">In Protobuf a data type is either a **`message`** (we can send it) or an **`enum`**.
```

Comment:
I would say `In a majority of cases, ...`. Because there is also options themselves. Eventually, the readers might want to have a look a `options.proto` which they imported for so many times. Each structure starts with `extends ...Options` in it.

Reply:
I want to keep it simple for the novice readers. That's why I make it straight as much as possible. Also, options are meta-data. So, an option type is meta-data type.